components/mpas-ocean/cime_config/buildnml

```diff
@@ -277,8 +278,8 @@ def buildnml(case, caseroot, compname):
         if ocn_ic_mode == 'spunup':
             ic_date = '230220'
             ic_prefix = 'mpaso.ECwISC30to60E2r1.rstFromG-anvil'
-
-
+        if ocn_ismf == 'data':
+            data_ismf_file = 'prescribed_ismf_adusumilli2020.ECwISC30to60E2r1.230429.nc'
```

Comment:
I need to add this data file for SORRM.  I think that's the only other mesh we need to support for now.